package.json

```diff
@@ -22,53 +22,52 @@
   },
   "homepage": "https://github.com/blackbaud/skyux-http#readme",
   "peerDependencies": {
-    "@angular/common": ">=4.3.6",
-    "@angular/core": ">=4.3.6",
-    "@blackbaud/auth-client": "^2.17.1",
+    "@angular/common": ">9.0.0-rc < 10.0.0",
```

Comment:
Do we know for sure `10.x` wouldn't supported?  I'm not necessarily opposed, but being explicit about not supporting future versions is a departure in our current procedures.

Reply:
Good question. Is it better to support all future versions or only a specific known subset? I've seen both setups in various packages. If SKY UX's stance is to support all future versions, I can change it.

Reply:
My vote would be to support it until we know it doesn't work.  Given our frequency is slower than that of Angular, that has bought us a few extra major versions in the past.